File: src/main/java/no/nav/personopplysninger/features/endreopplysninger/domain/adresse/EndringPostboksadresse.java
package no.nav.personopplysninger.features.endreopplysninger.domain.adresse;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import no.nav.personopplysninger.features.endreopplysninger.domain.Endring;

@JsonIgnoreProperties(ignoreUnknown = true)
public class EndringPostboksadresse extends Endring<EndringPostboksadresse> {
    private Postboksadresse innmeldtEndring;

    public Postboksadresse getInnmeldtEndring() {
        return innmeldtEndring;
    }
}

File: src/main/kotlin/no/nav/personopplysninger/features/endreopplysninger/domain/adresse/EndringPostboksadresse.kt
package no.nav.personopplysninger.features.endreopplysninger.domain.adresse

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import no.nav.personopplysninger.features.endreopplysninger.domain.Endring

@JsonIgnoreProperties(ignoreUnknown = true)
class EndringPostboksadresse : Endring<EndringPostboksadresse>() {
    val innmeldtEndring: Postboksadresse? = null
}
